src/core-styling/_typography-heading.scss

```diff
@@ -0,0 +1,67 @@
+@forward "typography-global";
+
+@use "sass:list";
+@use "sass:map";
+@use "settings/font";
+@use "baseline";
+@use "typography-global" as global-typography;
+
+$_h1-line-height: baseline.$unit-medium;
+$_h2-line-height: (baseline.$unit-small + baseline.$unit-smallish) / 2;
+$_default_line-height: baseline.$unit-small;
+$_font-family: font.$secondary;
+
+@mixin h1-typography() {
```

Comment:
Wondering whether this should just be `h1`? The module is specifically about typography, so `heading-typography.h1-typography` seems a bit redundant.